source/MapSalesPanel.cpp

```diff
@@ -358,11 +363,15 @@ void MapSalesPanel::Draw(Point &corner, const Sprite *sprite, bool isForSale, bo
 		
 		DrawSprite(corner, sprite);
 		
-		const Color &textColor = *GameData::Colors().Get(isForSale ? "medium" : "dim");
+		const Color &mediumColor = *GameData::Colors().Get("medium");
+		const Color &dimColor = *GameData::Colors().Get("dim");
+		const Color &textColor = isForSale ? mediumColor : !storage.empty() ? Color::Combine(.5, mediumColor, .5, dimColor) : dimColor;
```

Comment:
Sorry, changed. I also incorporated the change to the condition. I didn't incorporate the rename of `fooColor` to just `foo` since `textColor` is used elsewhere in the file and I wanted to be consistent.

Thanks for the review!